README.md

```diff
@@ -280,6 +280,72 @@ SLF4J Logger.
   metricRegistry.addReporter(reporter);
 ```
 
+#### Histograms
+
+In the current implementation, clients must define the distribution buckets and associate them in the reporter with the name of the metric to be histogrammed.
+
+There two types of distribution buckets available:
+- `DistributionBucket` represented by a primitive `long` array.
+- `DoubleValuedDistributionBucket` represented by a primitive `double` array
+  
+##### DistributionBucket
+Used to represent the distribution of a integer value. For example time spent in milliseconds or size of a messaging queue.
+
+For a given distribution array: [0, 10, 100, 500, 1000], the buckets would be like:
+* [0,10) for values 0-9
+* [10,100) for values 10-99
+* [100,500) for values 100-499
+* [500,1000) for values 500-999
+* overflow  for values  >= 1000
+* underflow for values  < 0
+
+```Java
+  String metricId = "latency";
+  DistributionBucket distributionBucket = new DistributionBucket(new long[]{0, 10, 50, 100});
+
+  SLF4JReporter reporter =
+      SLF4JReporter.builder().withName("metrics")
+          .addHistogram(metricId, distributionBucket)    // add histogram for metric with id "latency"
+          .build();
+    
+  String[] tagset = new String[] {"method", "GET", "resource", "metrics", "status", "200"};
+
+  Timer timer = metricRegistry.timer(metricId);    // creates a timer metric with id "latency"
+
+  long start = Timer.start();
+  // doSomething();
+  timer.stop(start, tagset); // records the latency and the distribution.
+```
+
+##### DoubleValuedDistributionBucket
+Used to represent the distribution of a floating point value. For example ads auction price.
```

Comment:
Would it be more precise to say, `double-precision floating point values`?

Reply:
yes.